collector/benchlib/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+name = "benchlib"
+version = "0.1.0"
+edition = "2021"
+description = "Library for defining and measuring benchmarks of Rust code"
```

Comment:
This is the only comment explaining what benchlib is. Maybe add a slightly more detailed explanation in one of the `.rs` files?

Reply:
Good idea, I added a comment to the `lib.rs` file.